Modules/core/Model/CoreConfig.php

```diff
@@ -70,9 +70,9 @@ public function isKey($key, $id_space) {
      * @param string $id_space
      */
     private function loadParams($id_space) {
-        if(CoreConfig::$params != null) {
+        /* if(CoreConfig::$params != null) {
```

Comment:
$params is not a static variable?
Could explain test against null, to avoid reloading params from db if already loaded.

Linked to other space config issue, where, in addition to insert/update operation, static var should also be updated.

So not an issue to remove test, but would avoid a new db call (means that params should be loaded when accessing a space...)